Sources/FoundationEssentials/ProcessInfo/ProcessInfo.swift

```diff
@@ -75,6 +75,16 @@ final class _ProcessInfo: Sendable {
             for env in environments {
                 let environmentString = String(cString: env)
 
+#if os(Windows)
+                // Windows GetEnvironmentStringsW API can return
+                // magic environment variables set by the cmd shell
+                // that starts with `=`
+                // We should exclude these values
+                if environmentString.hasPrefix("=") {
```

Comment:
Somewhat of a nit: while it's unlikely, it's possible that one of these magic variables could start with a grapheme extend character and this check would fail because this is checking based on the character view. Should we consider using `environmentString.utf8.first == UInt8(ascii: "=")` here to ensure that regardless of what follows the `=`, we catch anything that begins with the `=` scalar?

Reply:
No that's a good point! I love defensive programming. Thanks!!